packages/abi-gen-templates/contract.py.handlebars

```diff
@@ -0,0 +1,44 @@
+"""Generated wrapper for {{contractName}} Solidity contract."""
+
+from typing import Optional, Tuple, Union
+
+from hexbytes import HexBytes
+from web3.datastructures import AttributeDict
+from web3.providers.base import BaseProvider
+
+from zero_ex.contract_artifacts import abi_by_name
+
+from ._base_contract_wrapper import BaseContractWrapper
+from .tx_params import TxParams
+
+
+class {{contractName}}(BaseContractWrapper):
+    """Wrapper class for {{contractName}} Solidity contract."""
+
+    def __init__(
+        self,
+        provider: BaseProvider,
+        account_address: str = None,
+        private_key: str = None,
+    ):
+        """Get an instance of wrapper for smart contract.
+
+        :param provider: instance of :class:`web3.providers.base.BaseProvider`
+        """
+        super({{contractName}}, self).__init__(
+            provider=provider,
+            account_address=account_address,
+            private_key=private_key,
+        )
+
+    def _{{contractName}}(self, token_address):
```

Comment:
Instead of passing in the contract address here, let's require it at instantiation and store it as an instance variable. That way, one doesn't need to supply it with every method invocation.

Reply:
Also, why not just name this internal method `_getContractInstance`. No need to re-name for every contract, no?